### DISCUSSION TITLE
Firefox extension

### DISCUSSION THREAD

**Discussion by fractalf** [josdejong/svelte-jsoneditor#327] [Show and tell]
Hi!
Just wanted to say I made a Firefox extension based on this awesome project if anyone's interested :) 
Thanks a lot!

* https://github.com/fractalf/json-editor-ex
* https://addons.mozilla.org/en-US/firefox/addon/json-editor-ex/

**Comment by josdejong** [comment]
Thanks for sharing @fractalf , and thanks for keeping attribution in place 👍 .
